### PR TITLE
[feat] added Single Video Page

### PR DESCRIPTION
**Pull request by omjadhav85**
Features added:
1. Single Video View page with a video player to watch the video
2. Responsive design
3. Minor UI changes/corrections

### CODE REVIEW

File: src/pages/SingleVideoPage/VideoDetails.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { MdPlaylistPlay, MdThumbUp, MdWatchLater } from "react-icons/md";
+
+export const VideoDetails = ({ videoDetails }) => {
+  const { title, dateAdded, description, views, channel } = videoDetails;
+
+  return (
+    <div className="video-details">
+      <div className="h3">{title}</div>
```
Comment (junaidshaikh-js) [inline src/pages/SingleVideoPage/VideoDetails.js:8]: You can use semantic heading tags instead of `div`

File: src/utils/videoServerCalls.js [hunk 2]
```diff
@@ -8,3 +8,13 @@ export const getAllVideos = async (setVideos) => {
     console.error("Error getting all videos");
   }
 };
+
+export const getVideo = async (videoId, setVideoDetails) => {
+  try {
+    const res = await axios.get(`/api/video/${videoId}`);
+    setVideoDetails(res.data.video);
+    console.log("Single Video Data: ", res);
```
Comment (junaidshaikh-js) [inline src/utils/videoServerCalls.js]: clear console logs

**Review by junaidshaikh-js — COMMENTED** [review]
Great work done! 

Suggested some code refractor, everything else looks clean and good.

**Review by hinanshisuthar — COMMENTED** [review]
Great work done. Good to merge!

**Comment by omjadhav85** [comment]
Thanks Hinanshi and Junaid! Merging after 2 reviews

**Pull request merged**